prow/manifests/overlays/metal3/plugins.yaml

```diff
@@ -154,5 +155,7 @@ override:
   allow_top_level_owners: true
 
 triggers:
-- # Enable prow to trigger GitHub workflows.
+- repos:
+  - metal3-io
```

Comment:
Can you put entire org there? In test-infra there is always org/repo pair, and the config is named `repos:` which everywhere else means specfic repository?

/hold

Reply:
Ah, nevermind. It specifically says "or only org" in the example config.

/unhold
/lgtm